### ISSUE TITLE
Unable to include HTML in markup using tickmarks when it includes the Content tag

### ISSUE THREAD

**Issue by tomcam** [vuejs/vuepress#58]
I am trying to write up the process of creating .a custom theme.  When I tried to use this text in my markup:

```
<template>
  <div class="theme-container">
    <Content/>
  </div>
</template>
```

I get this cool thing where the content of my entire .md page gets recursively copied in a cool mirror-in-a-mirror effect. Unfortunately I do not like that cool effect in my notes.

Note that if instead if tickmarks I indent with 4 spaces instead of the 3 tickmarks this problem does away. But it's annoying.

Bonus question: I suspect "theme-container" is a placeholder for a class name but I don't know what it is.

**Comment by yyx990803** [comment]
Please provide a reproduction, the description is kinda vague

**Comment by yyx990803** [comment]
Btw your case is probably fixed in ac4acab. Let me know if your issue persists in the latest version.
